subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/BuildExperimentRunner.java

```diff
@@ -85,6 +87,18 @@ public void run(BuildExperimentSpec experiment, MeasuredOperationList results) {
         }
     }
 
+    private void configureCompilerDaemonAndTestWorkerMemory(BuildExperimentSpec experiment, File workingDirectory) {
```

Comment:
Makes sense. I now only added properties to `project-with-source`. Since I already changed tests in https://github.com/gradle/gradle/pull/1444, I will do the memory adjustment in the tests (if needed) there.